src/core_ocean/shared/mpas_ocn_tracer_hmix_redi.F

```diff
@@ -221,6 +223,11 @@ subroutine ocn_tracer_hmix_Redi_tend(meshPool, layerThickness, layerThicknessEdg
             r_tmp = dvEdge(iEdge)/dcEdge(iEdge)
             coef = dvEdge(iEdge)
 
+            k = minLevelEdgeBot(iEdge)
+            kappaRediEdgeVal = 0.25_RKIND*(RediKappaScaling(k, cell1) + RediKappaScaling(k, cell2) + &
+                                           RediKappaScaling(k + 1, cell1) + RediKappaScaling(k + 1, cell2))* &
+                               0.25_RKIND*(RediKappaSfcTaper(k, cell1) + RediKappaSfcTaper(k, cell2) + &
+                                           RediKappaSfcTaper(k + 1, cell1) + RediKappaSfcTaper(k + 1, cell2))
             k = 1
```

Comment:
why not minLevelEdgeBot here?

Reply:
I duplicated these lines first with `k=minLevelEdgeBot` and then `k=1` because I found that this was needed to preserve bit-for-bit results in standalone cases with ecosystem tracers. I can try removing these lines again after the following changes and see if I get the same result.

Reply:
I see, thanks for the explanation.  I doubt my suggestions below will help though.  I don't believe they will make a difference if minLevelCell(:) = 1

Just for my understanding L226-230 are extraneous right now correct?

Reply:
Yeah. I think it comes down to the relative importance of having clean code (delete L231-235) and retaining bit-for-bit results in ecosystem tracers cases (keep L231-235). Maybe @maltrud has an opinion on this?